### ISSUE TITLE
Next release of 5.X gpdb?

### ISSUE THREAD

**Issue by Daders** [greenplum-db/gpdb#10597]
Hello.
We are need a solution for one issue, which already in 5X_STABLE branch. And prefer to use released tags for production servers.
Could you give any insights about next release date of 5.X gpdb? Is it planning soon?

**Comment by kmacoskey** [comment]
Howdy @Daders . There is currently no specific date set for the next Greenplum 5 release. 

The major version is in a "Maintenance Mode" of sorts such that we do not plan to have any more Minor (feature) releases. The next Maintenance release, 5.28.1, will occur whenever there is a sufficient number of bug fixes or enough pressure to warrant a new Maintenance release.

What is the specific change in the 5X_STABLE branch that you need in the next release Maintenance release?

**Comment by Daders** [comment]
> What is the specific change in the 5X_STABLE branch that you need in the next release Maintenance release?

It was discussed in https://github.com/greenplum-db/gpdb/issues/10374 and fixed at gporca 3.106.0.
```
30559 - Query Optimizer
Queries that contain an IN clause with a large number of constants took a long time 
to generate a query plan.Most of the time was spent estimating the cardinality 
of the IN clause predicate. The cardinality estimation algorithm has been enhanced
and significantly reduces the cardinality estimation time for the specified type of query.
```

**Comment by kmacoskey** [comment]
@Daders The next maintenance release of Greenplum 5 was cut last night - 5.28.1. This includes the related commits.
